conda-recipes/multicoretsne-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: multicoretsne
+  version: {{ 0.1 }}
+
+source:
+  url: https://files.pythonhosted.org/packages/2d/e8/2afa896fa4eebfa1d0d0ba2673fddac45582ec0f06b2bdda88108ced5425/MulticoreTSNE-0.1.tar.gz
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
```

Comment:
For the compilers, add
```
   - {{ compiler('c') }}
   - {{ compiler('cxx') }}
   - cmake
   - make
```

Reply:
I just updated the recipe accordingly, and remove the yum install command from build_steps.sh